py14/tests/test_transpiler.py

```diff
@@ -76,7 +76,7 @@ def test_print_program_args():
     )
     cpp = transpile(source)
     assert cpp == parse(
-        "int main(int argc, char ** argv) {",
+        "int main(int argc, char * const argv[]) {",
```

Comment:
This is not C99 - it is C++14

Reply:
I looked for a reference for C++14 signature. This is what I found:

https://en.cppreference.com/w/cpp/language/main_function

Doesn't have a const on argv. Otherwise looks good.

Reply:
Alright I will remove this bit in the interests of getting traction.